src/index.html

```diff
@@ -12,7 +13,40 @@
       href="styles/index.scss"
```

Comment:
You should link a CSS file instead of SCSS. Browsers can't interpret SCSS files. It should be compiled into CSS.